recipes-extended/xen/xen-ocaml-libs.bb

```diff
@@ -86,7 +86,14 @@ do_compile() {
     oe_runmake -C tools/libs subdir-all-toollog
     oe_runmake -C tools/libs subdir-all-call
     oe_runmake -C tools/libs subdir-all-hypfs
+    oe_runmake -C tools/libs subdir-all-foreignmemory
+    oe_runmake -C tools/libs subdir-all-gnttab
+    oe_runmake -C tools/libs subdir-all-devicemodel
+    oe_runmake -C tools/libs subdir-all-evtchn
+    oe_runmake -C tools/libs subdir-all-ctrl
+    oe_runmake -C tools/libs subdir-all-store
     oe_runmake -C tools subdir-all-include
+    oe_runmake -C tools/libs subdir-all-guest
     oe_runmake LDLIBS_libxenctrl='-lxenctrl' \
```

Comment:
The xen libxl ocaml bindings were removed:
> commit e51d31f79edc10f8d622e7f5bf4b935af6d4618f
Author: Andrew Cooper <andrew.cooper3@citrix.com>
Date:   Thu Feb 9 15:25:10 2023 +0000
>
>    tools/ocaml: Drop libxl and xentoollog bindings
>    
>    There are significant issues with these bindings (they segfault for starters)
    and their companion half in Xapi was deleted in 2018
>    
>      https://github.com/xapi-project/xen-api/commit/203292ebe0c487d7ae4adb961a6d080f4fbe933d
>    
>    owing to there having been no development of these bindings since 2014.
>    
>    In the unlikely event that we'd want to reinstate them, they'd need reworking
    basically from scratch anyway.
>    
>    Signed-off-by: Andrew Cooper <andrew.cooper3@citrix.com>
>    Acked-by: Christian Lindig <christian.lindig@citrix.com>

so instead of adding all the new oe_runmake calls, just remove the long oe_runmake subdir-all-light call.

Reply:
Done and done.